app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def show
+    @tweet = Tweet.find_by(id: params[:id])
```

Comment:
find_byだとnilが返り、間違ったparams[:id]でもエラーが起きないため
findのほうがよいでしょう。

```suggestion
    @tweet = Tweet.find(params[:id])
```